Core1Platform/AdministratingCore1/05_EngineRoom.md

```diff
@@ -0,0 +1,239 @@
+
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened. The current workspace is overlapped.
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the [CLEAR TENANT CACHE] button top right.   
+   The following warning message is displayed:   
+
+   ![Warning message](../../Assets/Screenshots/Core1Platform/AdministratingCore1/ClearCacheWarning.png "[Warning message]")
+
+3. Click the [ABBRECHEN] (Cancel) button if you want to cancel the action. 
+
+4. If desired, click the [FORTFAHREN] (Continue) button to clear the cache.   
+   The cache is cleared. Afterwards, it will be rebuilt again. The processes on the instance are significantly slowed down.
+
+
+
+## Check warnings and notices
+
+The platform console provides instance-relevant notices and warning. In opposite to the notifications (see [Check notifications](../UsingCore1/03_GeneralUIFunctions.md#check-notifications "Check notifications")), these notices and warnings are targeted for developers.
+
+
+#### Procedure
+
+*Actindo Core1 Platform > Any workspace*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened. The current workspace is overlapped.
+
+   ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+2. Click the [PLATTFORM-KONSOLE] (Platform console) button.   
+   The *Platform Console* window is opened.
+
+   ![Platform console](../../Assets/Screenshots/Core1Platform/AdministratingCore1/EngineroomPlatformConsole.png "[Platform console]")
+
+3. Check the messages.
+
+
+
+## Check vCores
+
+Your company has purchased a certain number of vCores assigned the main account. vCores (CPU core voltage) define the computing power of an instance for the execution of parallel processes as well as database queries. Idle times are not considered, for example calls waiting for a response from external service providers.
+If an instance has not enough vCore capacity, the processor then starts to run the processes and database queries one after the other instead of running them in parallel. This means that the processing speed is slowed down significantly if the number of vCores assigned is unsufficiently.  
+
+> [INFO] The vCores define the power for running parallel processes. They do not define the power for running a single process. Assigning more vCores will not speed up single processes!    
+
+If a sandbox is created, it is assigned a number of vCores from the main account. This reduces the number of vCores available for the main account. Note, the transfer of vCores from the main account to a sandbox may take some time (about 5 minutes).  
```

Comment:
Note, --> Note that... 
Bitte durchgängig prüfen

Reply:
done